staging/src/k8s.io/pod-security-admission/policy/checks.go

```diff
@@ -80,6 +82,9 @@ type CheckResult struct {
 	// - list specific invalid containers: "container1, container2"
 	// - list specific non-default capabilities: "CAP_NET_RAW"
 	ForbiddenDetail string
+	// ErrList must be set if Allowed is false.
```

Comment:
Is `ErrList ` required?

Reply:
What do you mean ?

Reply:
Fixes, it should only be set if Allowed is false, and is optional.